src/notify.js

```diff
@@ -28,6 +28,8 @@ function Notify(title, options = {}) {
         notifyError = null,
         closeOnClick = false,
         timeout = null,
+        swUrl = null,
```

Comment:
Actually, this is the method that I seek. To be honest I hadn't even heard about Service Workers or Web Workers before trying the Notifications API. It worked perfectly on the desktop, but when I tried it in my Android phone, I was surprised to realize it's not working. Connecting the phone to my PC and checking the console log on the desktop I ran into Service Workers...

In my project I have a web socket listening, and if there is a certain event from the server, I want to notify the user... I'm not using the Push API at all. I have no clue what most users looking at the Notifications API are using, but at least from the people around me, no one seems to be aware of Service Workers, etc. 

I actually think that this use case is far more useful, since inside the Service Worker, there is no API ambiguity, while in normal web pages there are two different APIs supported differently across browsers.